speaker_load_mobile.js

```diff
@@ -0,0 +1,158 @@
+const speakerSection = document.querySelector('#mobile_skpeakers');
```

Comment:
- [ ] According to the project requirement "speakers"  section should be created dynamically in JavaScript. For that 
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

```
{
  name: "Specaker name ", 
  role: "Java developer facebook", 
  description: "Speaker description ...", 
  photo: "image_url_here"
}

```